### PR TITLE
closes #76 fix adsl bugs

### LINKED ISSUE #76: Bug in ADaM ADSL example
The following result from the ADaM ADSL webpage doesn't look as expected.
image

By the way, there is a typo in the TOC:
image

There are two typos in the following comment:
image

### PR DESCRIPTION
**Pull request by rossfarrugia**
## Pull Request

Fix some minor ADSL bugs spotted by Stefan

Before you submit your pull request, take a look at the following checklist. Many thanks for your contribution! 

- [x] **Title:** Place `Closes #<insert_issue_number>` at the beginning of your PR title. Use the Edit button in the top-right if you need to update.
- [x] **Linked Issue:** Ensure the related issue is linked in the "Development Section" on the right-hand side.
- [x] **First Contribution:** If this is your first contribution, add yourself to the `DESCRIPTION` file.
- [x] **Impact on Examples:** If your updates impact any examples, review locally for warnings or errors in the impacted example pages.
- [x] **Merge Conflicts:** Developers should address any merge conflicts and merge upon successful review.
- [x] **New Packages:** If new packages were used, ensure they are included in the `DESCRIPTION` file's `Imports` section.
- [x] **Updated Examples:** If you added or updated an example, ensure it runs on the latest CRAN release versions of all packages used.
- [x] **Testing Instructions:** Provide instructions on how to test the code if necessary.

### CODE REVIEW

File: adam/adsl.R [hunk 1]
```diff
@@ -9,39 +9,36 @@ library(tidyr)
...
+metacore <- metacore %>%
    select_dataset("ADSL")
 
 ## ----r------------------------------------------------------------------------
 metacore$ds_vars
 
-## ----r, error=TRUE------------------------------------------------------------
-build_from_derived(metacore, list(), predecessor_only = FALSE)
-
 ## ----r demographcis-----------------------------------------------------------
```
Comment (bundfussr) [inline adam/adsl.R:25]: Typo: "demographcis" instead of "demographics"

**Review by bundfussr — COMMENTED** [review]
Updates look good to me.

I've added just a very minor comment and I wonder if the files should be run through `styler::style_file()`.

**Comment by bms63** [comment]
> Updates look good to me.
> 
> I've added just a very minor comment and I wonder if the files should be run through `styler::style_file()`.

Since we are building up more of a code base - I propose we set up styling and spelling github actions now.

**Comment by rossfarrugia** [comment]
Good idea, I added https://github.com/pharmaverse/examples/issues/82 and will see if anyone wants to take it on.

**Review by bundfussr — APPROVED** [review]

**Comment by rossfarrugia** [comment]
@bundfussr now granted

**Pull request merged**